lib/config-validator.ts

```diff
@@ -20,6 +20,7 @@ async function validate(
   desc: string,
   config: RenovateConfig,
   strict: boolean,
+  isGlobalConfig: boolean,
   isPreset = false
```

Comment:
careful, when you end in two booleans then it could be easy to miss some migrations here. Look at line 131 for example

Reply:
```suggestion
  isGlobalConfig: boolean,
  isPreset: boolean
```

force parameter to not get into migration trouble